Tasks/DeployVisualStudioTestAgent/DeployTestAgent.ps1

```diff
@@ -14,10 +18,20 @@ param(
 $logonAutomatically = $runAsProcess
 $disableScreenSaver = $runAsProcess
 
+Function CmdletHasMember($memberName) {
+    $cmdletParameter = (gcm Invoke-DeployTestAgent).Parameters.Keys.Contains($memberName) 
+    return $cmdletParameter
+}
+
+Function Get-PersonalAccessToken($vssEndPoint) {
+    return $vssEndpoint.Authorization.Parameters.AccessToken
+}
+
 Write-Verbose "Entering script DeployTestAgent.ps1"
-Write-Verbose "testMachineGroup = $testMachineGroup"
+Write-Verbose "testMachineInput = $testMachineGroup"
+Write-Verbose "WinRmProtocal = $winRmProtocol"
 Write-Verbose "resourceFilteringMethod = $resourceFilteringMethod" -Verbose
-Write-Verbose "testMachines = $testMachines"
+Write-Verbose "filter testMachines = $testMachines"
```

Comment:
The initial parameter name is went wrong. testMachineGroup = input of test machines
testMachines = filter value. To make it easy, I added that